imports/plugins/included/wallet/client/checkout/wallet.js

```diff
@@ -0,0 +1,192 @@
+import {
+  Meteor
+} from "meteor/meteor";
+import {
+  Template
+} from "meteor/templating";
+import {
+  AutoForm
+} from "meteor/aldeed:autoform";
+import {
+  $
+} from "meteor/jquery";
+import {
+  Reaction
+} from "/client/api";
+import {
+  Cart,
+  Shops,
+  Packages,
+  Accounts
+} from "/lib/collections";
+import {
+  Wallet
+} from "../../lib/api";
+import {
+  WalletPayment
+} from "../../lib/collections/schemas";
+
+import "./wallet.html";
+
+let submitting = false;
+
+/**
+  *
+  * @param {any} template
+  * @param {any} buttonText
+  *
+  * @returns {Object} template
+  */
+function uiEnd(template, buttonText) {
+  template.$(":input").removeAttr("disabled");
+  template.$("#btn-complete-order").text(buttonText);
+  return template.$("#btn-processing").addClass("hidden");
+}
+
+/**
+  *
+  * @param {any} errorMessage
+  *
+  * @returns {Object} payment alert object
+  */
+function paymentAlert(errorMessage) {
+  return $(".alert").removeClass("hidden").text(errorMessage);
+}
+
+/**
+  *
+  * @returns {Object} alert object
+  */
+function hidePaymentAlert() {
+  return $(".alert").addClass("hidden").text("");
+}
+
+/**
+  *
+  * @param {any} error
+  *
+  * @returns {Object} alert object
+  */
+function handleWalletSubmitError(error) {
+  const serverError = error !== null ? error.message : "undefined";
+  if (serverError) {
```

Comment:
I don't see a difference between this line and line 74. The only line necessary in this function is
```js
return paymentAlert(`Oops! ${error}, null, 4);
```